mino/minows/key/mod.go

```diff
@@ -0,0 +1,63 @@
+package key
+
+import (
+	"crypto/rand"
+	"github.com/libp2p/go-libp2p/core/crypto"
+	"go.dedis.ch/dela/core/store/kv"
+	"golang.org/x/xerrors"
+)
+
+// Storage provides persistent Storage for private keys.
+type Storage struct {
+	bucket []byte
+	db     kv.DB
+}
+
+// NewStorage creates a new Storage for private keys.
+func NewStorage(db kv.DB) *Storage {
+	return &Storage{
+		bucket: []byte("keys"),
```

Comment:
This bucket might create plenty of conflicts with other parts of the application making use of the database, as the name is neither particularly rare nor namespaced in any way.

On a separate note, I can't help but think that the database should probably offer a centralized utility for namespacing buckets.

Reply:
Good catch - I'll change it to `minows_keys`, which should be unique enough.

I thought there was some namespacing, or was that only in the key/value storage part of Dela?

Reply:
The namespacing does indeed happen, but at a higher level, on the key-value storage abstraction exposed to the contracts.